tests/integration/ef_tests/test_ef_blockchain_tests.py

```diff
@@ -41,8 +41,6 @@ async def setup_test_state(
             storage_entries = (
                 (("is_initialized",), 1),
                 (("evm_address",), evm_address),
-                (("evm_to_starknet_address", evm_address), starknet_address),
-            ) + (
```

Comment:
should this + sign be removed as well?

Reply:
traumatized by python, i didnt know what it did

Reply:
it basically combines the base tuples (left hand side) with either the necessary eoa or contract tuples!